demisto_sdk/commands/setup_env/setup_environment.py

```diff
@@ -0,0 +1,406 @@
+    create_integration_instance,
+)
+
+json5 = JSON5_Handler()
+json = JSON_Handler()
+
+
+class IDE(Enum):
+    VSCODE = "vscode"
+    PYCHARM = "pycharm"
+
+
+IDE_TO_FOLDER = {IDE.VSCODE: ".vscode", IDE.PYCHARM: ".idea"}
+
+
+def get_integration_params(project_id: str, secret_id: str) -> dict:
+    """This function retrieves the parameters of an integration from Google Secret Manager
+    *Note*: This function will not run if the `DEMISTO_SDK_GCP_PROJECT_ID` env variable is not set.
+
+    Args:
+        project_id (str): GSM project id
+        secret_id (str): The secret id in GSM
+
+    Returns:
+        dict: The integration params
+    """
+    # Create the Secret Manager client.
+    client = secretmanager.SecretManagerServiceClient()
+
+    # Build the resource name of the secret version.
+    name = f"projects/{project_id}/secrets/{secret_id}/versions/latest"
+
+    # Access the secret version.
+    try:
+        response = client.access_secret_version(name=name)
+    except google.api_core.exceptions.NotFound:
+        logger.warning("The secret is not found in the secret manager")
+        return {}
+    except google.api_core.exceptions.PermissionDenied:
+        logger.warning(
+            "Insufficient permissions for gcloud. If you have the correct permissions, run `gcloud auth application-default login`"
+        )
+        return {}
+    except Exception:
+        logger.warning(
+            f"Failed to get secret {secret_id} from secret manager, skipping"
+        )
+        return {}
+    # Return the decoded payload.
+    payload = response.payload.data.decode("UTF-8")
+    return json5.loads(payload).get("params")
+
+
+def add_init_file_in_test_data(integration_script: IntegrationScript):
+    if (integration_script.path.parent / "test_data").exists():
+        (integration_script.path.parent / "test_data" / "__init__.py").touch()
+
+
+def configure_dotenv():
+    """
+    This functions configures the .env file located with PYTHONPATH and MYPYPATH
+    """
+    dotenv_path = CONTENT_PATH / ".env"
+    env_vars = dotenv.dotenv_values(dotenv_path)
+    env_vars["PYTHONPATH"] = ":".join([str(path) for path in PYTHONPATH])
+    env_vars["MYPYPATH"] = ":".join([str(path) for path in PYTHONPATH])
+    for key, value in env_vars.items():
+        if not value:
+            continue
+        dotenv.set_key(dotenv_path, key, value)
+
+
+def configure_vscode_settings(
+    ide_folder: Path, integration_script: IntegrationScript, interpreter_path: Path
+):
+    shutil.copy(Path(__file__).parent / "settings.json", ide_folder / "settings.json")
+    with open(ide_folder / "settings.json") as f:
+        settings = json5.load(f)
+
+    settings["python.defaultInterpreterPath"] = str(interpreter_path)
+    settings["python.testing.cwd"] = str(integration_script.path.parent)
+    with open(ide_folder / "settings.json", "w") as f:
+        json.dump(settings, f, indent=4)
```

Comment:
not json5?

Reply:
It doesn't really matter. It's just a normal JSON, we don't need json5 here